charts/argo-cd/values.yaml

```diff
@@ -9,6 +9,11 @@ fullnameOverride: ""
 # -- Override the Kubernetes version, which is used to evaluate certain manifests
 kubeVersionOverride: ""
 
+## Custom resource configuration
+crds:
+  # -- Install and upgrade CRDs
+  install: true
```

Comment:
This is same as [argo-events](https://github.com/argoproj/argo-helm/blob/main/charts/argo-events/values.yaml#L18-L21).